doc/source/getting_started/first_steps.rst

```diff
@@ -1,8 +1,9 @@
-###############
-Getting started
-###############
+.. _first_steps:
+
+##################
+First steps
+##################
 
-Welcome to the ODL getting started guide.
 This guide is intended to give you a simple introduction to ODL and how to work with it.
 If you need help with a specific function you should look at the `ODL API reference <https://odlgroup.github.io/odl/odl.html>`_.
 
```

Comment:
Also found while reading this: We use conjugate_gradient_normal, but the operator is self adjoint and we could use conjugate_gradient. Do we want to fix this?